### DISCUSSION TITLE
UQL unable to get query from nesting json file

### DISCUSSION THREAD

**Discussion by seasoncool** [grafana/grafana-infinity-datasource#534] [Q&A] [answered]
Hi all, 

I want query **id** and **System.State** fields with UQL, and I have tried few formats with no luck :( . please help me out, thanks.

![image](https://user-images.githubusercontent.com/6467295/220238241-3cadf0fd-fce8-44a4-950c-795f8e3e0074.png)

json:

{
	"value": [{
			"id": 100,
			"rev": 1,
			"fields": {
				"System.WorkItemType": "Abnormal",
				"System.State": "Done",
				"System.Reason": "Work finished"
			},
			"url": "https://www.google.com"
		},
		{
			"id": 101,
			"rev": 12,
			"fields": {
				"System.WorkItemType": "Abnormal",
				"System.State": "Done",
				"System.Reason": "Work finished"
			},
			"url": "https://www.github.com"
		}
	]
}

**Comment by yesoreyeram** [comment, marked as answer]
what you wanted is to scope it to "value" so that UQL will treat only value inside that. 

```kusto
parse-json
| scope "value"
| project "id", "state"="fields['System.State']"
```

**Comment by seasoncool** [comment]
thanks ,fixed.
